workloads/network-perf-v2/run.sh

```diff
@@ -4,7 +4,10 @@ set -e
 source ./env.sh
 source ../../utils/common.sh
 
-curl -sS -L $NETPERF_URL | tar -xz
+# Download and extract k8s-netperf if we haven't already
+if [ ! -f "${NETPERF_FILENAME}" ]; then
```

Comment:
hm, could this cause issues if people are flipping versions of k8s-netperf?

Reply:
Gimme a `k8s-netperf --version` flag to work with?

Reply:
Fair ;)

Reply:
Done.